tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -53,6 +61,22 @@ final class BuildCommand extends CommandBase {
       if (!useRbe) '--no-rbe',
     ];
 
-    return runBuild(environment, build, extraGnArgs: extraGnArgs);
+    final Map<String, BuildTarget> allTargets = await findTargets(environment,
```

Comment:
This directory might not exist if GN hasn't had a chance to run yet. That is, if you have a fresh engine checkout and `out/` is empty, then `selectTargets` will fail.